Source/Core/Core/HW/DSP.cpp

```diff
@@ -397,8 +397,9 @@ void RegisterMMIO(MMIO::Mapping* mmio, u32 base)
 
           // TODO: need hardware tests for the timing of this interrupt.
```

Comment:
This TODO can probably be removed if the hardware test is added to the [hwtests](https://github.com/dolphin-emu/hwtests) repository, so its not lost track of

Reply:
IMO, it still needs a proper hardware test that measure the cycle it takes for this interrupt. Since my test is based on the libogc's DMA callback its result is biased. Moreover, Dolphin seems to execute instructions slightly faster than a real Wii.

The timing I chose for this interrupt is a "close-enough" value that reach the timing expected by the hardware test without taking into account Dolphin's speed. In sum, this test doesn't measure the cycles it takes but roughly approximates it.